src/ui/marker.js

```diff
@@ -364,16 +388,30 @@ export default class Marker extends Evented {
     }
 
     /**
-     * Returns the Popup instance that is bound to the Marker
+     * Returns the `Popup` instance that is bound to the `Marker`.
```

Comment:
```suggestion
     * Returns the {@link Popup} instance that is bound to the {@link Marker}.
```

Reply:
I'm loving the use of these internal links @danswick .